Tasks/Kubernetes/task.json

```diff
@@ -195,6 +200,53 @@
             "helpMarkDown": "Delete the secret if it exists and create a new one with updated values.",
             "groupName": "secrets"
         },
+        {
+            "name": "configMapName",
+            "type": "string",
+            "label": "ConfigMap Name",
+            "defaultValue": "",
+            "helpMarkDown": "Name of ConfigMap.",
+            "groupName": "configMaps"
+        },
+        {
+            "name": "forceUpdateConfigMap",
+            "type": "boolean",
+            "label": "Force update configmap",
+            "defaultValue": "true",
+            "helpMarkDown": "Delete the configmap if it exists and create a new one with updated values.",
+            "groupName": "configMaps"
```

Comment:
From the spec i can see that Secrets and Config map is combined. Did you talk to Atul.